ui/intermittent-failures/helpers.js

```diff
@@ -119,3 +119,15 @@ export const tableRowStyling = function tableRowStyling(state, bug) {
   }
   return {};
 };
+
+export const removePath = function removePath(line = '') {
+  const [a, testNameWithPath, ...rest] = line.split(' | ');
```

Comment:
These const's are not very clear - can you rename them? 

Also, since we want to remove all characters that are part of the path, it might make more sense to use `replace`.

Reply:
ok, I will do that.